db/migrations/postgres/000029_create_sidebar_channels.up.sql

```diff
@@ -0,0 +1,30 @@
+CREATE TABLE IF NOT EXISTS sidebarchannels (
+    channelid VARCHAR(26),
+    userid VARCHAR(26),
+    categoryid VARCHAR(128),
```

Comment:
Since we are adding it on the do block bellow, let's remove it from the create table.